pkg/audit/audit_segment.go

```diff
@@ -18,78 +17,64 @@ import (
 	"storj.io/storj/pkg/storage/meta"
 )
 
-// Audit to audit segments
-type Audit struct {
-	pointers pdbclient.Client
-	lastPath *paths.Path
-	mutex    sync.Mutex
-}
-
-// NewAudit creates a new instance of audit
-func NewAudit(pointers pdbclient.Client) *Audit {
-	return &Audit{
-		pointers: pointers,
-	}
-}
-
 // Stripe is a struct that contains stripe info
 type Stripe struct {
 	Index int
 }
 
 // NextStripe returns a random stripe to be audited
-func (audit *Audit) NextStripe(ctx context.Context) (stripe *Stripe, more bool, err error) {
-	audit.mutex.Lock()
-	defer audit.mutex.Unlock()
+func (auditor *Auditor) NextStripe(ctx context.Context) (stripe *Stripe, pointer *pb.Pointer, more bool, err error) {
```

Comment:
Should the pointer be a field of the Stripe type, instead of returning it as another return value. 

i.e. a Stripe is defined by:
 1) the segment it belongs to - this is the pointer
 2) its index within the segment